### ISSUE TITLE
Listing filters with translation separately?

### ISSUE THREAD

**Issue by tokyogeometry** [dtschump/gmic-community#36]
Hi, I'm just curious but can I have the filters with translation shown separately at the top or the bottom of the list in G'MIC window, as like:

![_2016-01-16_00-21-03](https://cloud.githubusercontent.com/assets/12570829/12357330/971de35c-bbea-11e5-9342-d7a43bc19619.jpg)

**Comment by dtschump** [comment]
It should be possible, but for now, there is a small bug in the update filter script that makes this not working as expected. I'll try to fix it as soon as possible.
That's why I reverted your translated section names in plain English instead of keeping them in Japanese.

**Comment by tokyogeometry** [comment]
OK, thank you!
